addons/crm_claim/views/res_partner_view.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0"?>
-<openerp>
+<odoo>
     <data>
         <!--  Partners inherited form -->
         <record id="view_claim_res_partner_info_form" model="ir.ui.view">
```

Comment:
change <field name="name">res.partner.claim.info.form</field> to 
<field name="name">res.partner.form.inherit.claim.info</field>